data/state/Talents.yaml

```diff
@@ -4,3 +4,11 @@ value:
   activeConfigID: 12345
   activeConfigID_comment: strictly speaking should be spec specific (nil if no spec
     selected)
+  pendingViewLoadoutLevel: null
+  pendingViewLoadoutLevel_comment: currently unused
+  pendingViewLoadoutSpecID: null
+  viewLoadoutDataImported: false
+  viewLoadoutDataImported_comment: true after ViewLoadout is called, reset on InitializeViewLoadout
+    currently used to generate warnings for bad usage. This deviates from how the
+    real client handles it!
+  viewLoadoutSpecID: null
```

Comment:
So... yaml "null" is weird. I'm not sure if it ends up as Lua nil. Maybe just have a comment field with all these fields documented. This situation needs to be better :-/

Reply:
I had `viewLoadoutSpecID:\n` before, but the auto reformatter changed it to this
I'm not a fan of leaving it out completely

Reply:
Okay we can leave it this way for now and iterate later.